server/lib/mix/tasks/minio_setup.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Mix.Tasks.Minio.Setup do
+  use Mix.Task
+
+  @shortdoc "Creates in Minio the required buckets for development and test"
```

Comment:
nitpick:
"Cretes the required buckets for dev/test in Minio" sounds more correct